### ISSUE TITLE
add ListOptions to ListWatched 

### ISSUE THREAD

**Issue by bradrydzewski** [google/go-github#238]
I'd like the ability to provide `ListOptions` when getting the list of watched repositories:

```diff
-func (s *ActivityService) ListWatched(user string)
+func (s *ActivityService) ListWatched(user string, opt *ListOptions)
```

I can have a pull request available next week

**Comment by willnorris** [comment]
sounds good to me

**Comment by sona-tar** [comment]
:+1:

**Comment by willnorris** [comment]
fixed in  c57bb02
